src/uct/api/uct.h

```diff
@@ -1059,6 +1065,32 @@ struct uct_iface_params {
 
     /* Time period between keepalive rounds */
     ucs_time_t                                   keepalive_interval;
+
+    /**
+     * Desired alignment for Active Messages on the receiver. Note that only
+     * data received in the UCT descriptor can be aligned (i.e.
+     * @a UCT_CB_PARAM_FLAG_DESC flag is provided in the Active Message
+     * handler callback). The provided value must be power of 2.
```

Comment:
but it is not. Typically we use `UCS_CACHE_LINE_SIZE` for aligning. But some transports align by user data, others (ib) align by header

Reply:
If user does not require a specific alignment, we can consider this as if the required alignment is 1.
the transport may have its own alignment but it would still be aligned to 1

Reply:
imo, this statement would be confusing

Reply:
can we say "the default value is 1, which means no specific alignment is requested"?

Reply:
fixed